functional_widget/lib/function_to_widget_class.dart

```diff
@@ -16,10 +16,13 @@ final _hookWidgetRef = refer('HookWidget', _kHookWidgetsPath);
 final _keyRef = refer('Key', _kFlutterWidgetsPath);
 final _buildContextRef = refer('BuildContext', _kFlutterWidgetsPath);
 
-String _toTitle(String string) {
-  return string.replaceFirstMapped(RegExp('[a-zA-Z]'), (match) {
+String _toTitle(String string, bool removeLeadingUnderscore) {
```

Comment:
I'd prefer having two separate functions:
- `toTitle`
- `toPublic`

Reply:
It's not create what `toPublic` means here. We only remove the first underscore, so it can still be private... `_toTitleRemoveLeadingUnderscore` would be a better name.

However, by adding a second function, we will also introduce bunch of `?` in the code. For example:

```
_toTitle(
              functionElement.name,
              annotation.removeLeadingUnderscore ??
                  _defaultOptions.removeLeadingUnderscore)```

would become

```
annotation.removeLeadingUnderscore ?? _defaultOptions.removeLeadingUnderscore ? _toTitleRemoveLeadingUnderscore(functionElement.name) : _toTitle(functionElement.name)
```

Pick your poison!

Reply:
Good point! You can leave it as is, then 😄